### ISSUE TITLE
[Linked List]: React - VO - Linked List titles are read out twice

### ISSUE THREAD

**Issue by annawen1** [carbon-design-system/carbon-for-ibm-dotcom#8134]
### Description

The Linked Lists are read out twice instead of once.
One time for the link list title, second time again + announcing that it is a link/interactive element.

https://user-images.githubusercontent.com/54281166/150828655-5000e87d-a857-441a-9660-91103b97471b.mov

### Component(s) impacted

Link List

### Browser

Safari

### Carbon for IBM.com version

1.30.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/react/?path=/story/components-link-list--default

### Package

@carbon/ibmdotcom-react

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by annawen1** [comment]
Realizing, those link list items in default are cards and the way cards are read out is consistent with what was reported here. So closing this as it doesn't seem like a bug.
